extension/content_script.js

```diff
@@ -503,12 +503,13 @@ function oneTimeSetup() {
 		// Listen for background.js toggle pings.
 		chrome.runtime.onMessage.addListener(
 			function(request, sender, sendResponse) {
-			    if (request.command === "toggleUI") {
-			    	toggleExtensionVisibility();
-			    }
+				if (request.command === "toggleUI") {
+					toggleExtensionVisibility();
+				}
```

Comment:
This must have come in when I was doing the merge conflict resolution or something, is this supposed to be here?

Reply:
The lines are supposed to be there.
I'm guessing you are talking about the tab? Hmmm I'm not sure - might be me accidentally tabbing.

Reply:
No just the lines in general; I'm not sure what happened, seems like an aberration

Reply:
if (request.command === "toggleUI") {
  toggleExtensionVisibility();
}

is the right logic here -- if that's what we're discussing. hmm

Reply:
Yeah I don't know why it was counted as being added in this pull request, if it is supposed to be there 
(That was original query; so I think answer is this is fine?)